DataFormats/Track/interface/alpaka/PixelTrackUtilities.h

```diff
@@ -0,0 +1,199 @@
+#ifndef DataFormats_Track_PixelTrackUtilities_h
+#define DataFormats_Track_PixelTrackUtilities_h
+
+#include "DataFormats/TrackerCommon/interface/SimplePixelTopology.h"
+#include "DataFormats/Track/interface/PixelTrackDefinitions.h"
+#include "DataFormats/Track/interface/PixelTrackLayout.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
```

Comment:
As far as I can tell, nothing in this file needs to be in `ALPAKA_ACCELERATOR_NAMESPACE`.